modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-a-file.adoc

```diff
@@ -68,9 +66,10 @@ data:
   settings.xml: __<base64 encoded data content here>__
 ----
 +
-This results in a file named `settings.xml` being mounted at the `{prod-home}/.m2/` path of all workspace containers.
-+
-The secret-s mount path can be overridden for specific components of the workspace using devfile. To change mount path, an additional volume should be declared in a component of the devfile, with name matching overridden secret name, and desired mount path.
+NOTE: The `data` of the {orch-name} secret can contain several items. The names of these items must match the desired file name mounted into the container.
+
+* You can override the mount path of the secret for a workspace component by editing the devfile as follows: Declare a component alias for the containers in the override path. Declare an additional volume for the component. Use the name of the secret as the name of the volume. Provide the new mount path for the secret as `containerPath`. See the following example.
+//"volume" here means storage volume? max-cx
```

Comment:
Find out before submitting the doc. Such comments in the file add confusion. (And yes, it pbbly means 'storage volume'.)

Reply:
@rkratky, OK. I reasoned these comments would be read and answered at the review of this merge request. And if not, I'd still be rewriting the whole file for the follow-on issue.